### ISSUE TITLE
Issue occur when I Start testing the project

### ISSUE THREAD

**Issue by csharp489** [fury-gl/fury#219]
When I start running the test, it gives me these errors .Please help me
code: 
`pytest -svv fury`
Output: 
```python
PS C:\Users\rishabh\FURY> pytest -svv fury                                                                                                                                          =============================================================================== test session starts ===============================================================================
platform win32 -- Python 3.6.2, pytest-5.3.2, py-1.8.1, pluggy-0.13.1 -- c:\python36\python.exe
cachedir: .pytest_cache
rootdir: C:\Users\rishabh\FURY
plugins: cov-2.8.1, forked-1.0.2, localserver-0.5.0, xdist-1.31.0
collected 29 items / 6 errors / 23 selected

===================================================================================== ERRORS ======================================================================================
___________________________________________________________________ ERROR collecting fury/tests/test_actors.py ____________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_actors.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_actors.py:11: in <module>
    from fury import actor, window
fury\actor.py:16: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
_________________________________________________________________ ERROR collecting fury/tests/test_interactor.py __________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_interactor.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_interactor.py:7: in <module>
    from fury import actor, window, interactor
fury\actor.py:16: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
__________________________________________________________________ ERROR collecting fury/tests/test_primitive.py __________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_primitive.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_primitive.py:3: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
_____________________________________________________________________ ERROR collecting fury/tests/test_ui.py ______________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_ui.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_ui.py:13: in <module>
    from fury import window, actor, ui
fury\actor.py:16: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
____________________________________________________________________ ERROR collecting fury/tests/test_utils.py ____________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_utils.py:12: in <module>
    from fury import actor, window, utils
fury\actor.py:16: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
___________________________________________________________________ ERROR collecting fury/tests/test_window.py ____________________________________________________________________
ImportError while importing test module 'C:\Users\rishabh\FURY\fury\tests\test_window.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
c:\python36\lib\site-packages\_pytest\python.py:492: in _importtestmodule
    mod = self.fspath.pyimport(ensuresyspath=importmode)
c:\python36\lib\site-packages\py\_path\local.py:701: in pyimport
    __import__(modname)
c:\python36\lib\site-packages\_pytest\assertion\rewrite.py:143: in exec_module
    exec(co, module.__dict__)
fury\tests\test_window.py:7: in <module>
    from fury import actor, window, io
fury\actor.py:16: in <module>
    import fury.primitive as fp
fury\primitive.py:6: in <module>
    from scipy.spatial import transform
E   ImportError: cannot import name 'transform'
================================================================================ warnings summary =================================================================================
c:\python36\lib\site-packages\win32\lib\pywintypes.py:2
  c:\python36\lib\site-packages\win32\lib\pywintypes.py:2: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp, sys, os

-- Docs: https://docs.pytest.org/en/latest/warnings.html
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 6 errors during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
========================================================================== 1 warning, 6 errors in 7.23s ===========================================================================
```

Please let me know why this error is occurring.

**Comment by Nibba2018** [comment]
Have you installed the required dependencies correctly?

**Comment by skoudoro** [comment]
Hi @csharp489,

I agree with @Nibba2018. It seems you have the wrong version of scipy

**Comment by csharp489** [comment]
I have created a first.py file with the code given by @Nibba2018 
![TestScipy](https://user-images.githubusercontent.com/42842260/77926843-39f90900-72c4-11ea-9a65-15a8ad448fa9.PNG)
@skoudoro @Nibba2018 please check the output.

**Comment by Nibba2018** [comment]
@csharp489  Try to bump up your *Python* version to `3.7` or *scipy* version to `1.4.1`.

**Comment by skoudoro** [comment]
No need to bump python version but we need to check for scipy and it's transform module. I'm still surprised

**Comment by skoudoro** [comment]
ok, I just checked,  `transform` module is available from Scipy 1.2. However, you have Scipy 1.1.0. So :

- you need to update you Scipy version
- we need to create a PR and update Scipy minimum version.

Thank you for the feedback.

**Comment by Nibba2018** [comment]
@skoudoro Yes scipy v1.2 has the required module. Should I make a quick PR to bump the scipy version?

**Comment by csharp489** [comment]
@skoudoro Thankyou for your instant reply .Now it has been fixed.
I have updated the Scipy module using this command
`pip install --upgrade scipy`

**Comment by csharp489** [comment]
It is running fine. 
![imageScipy](https://user-images.githubusercontent.com/42842260/77944801-31f99300-72dd-11ea-8f91-e63203a53e5d.PNG)

**Comment by skoudoro** [comment]
Great @csharp489!

> @skoudoro Yes scipy v1.2 has the required module. Should I make a quick PR to bump the scipy version?

yes, you can go ahead @Nibba2018. It should be a quick and fast PR to create and merge. 
Then we can close this issue.

**Comment by skoudoro** [comment]
fixed by #220, closing
